test-runtime/setup.js

```diff
@@ -17,7 +17,9 @@ module.exports = async function () {
 
   console.log(chalk.green('Setup Puppeteer'));
 
-  const browser = await puppeteer.launch();
```

Comment:
Without this, the puppeteer tests do not run inside Docker.
It is mentioned here: https://github.com/puppeteer/puppeteer/blob/main/docs/troubleshooting.md#setting-up-chrome-linux-sandbox

Reply:
https://github.com/puppeteer/puppeteer/blob/main/docs/troubleshooting.md#running-puppeteer-in-docker doesn't mention it. Are you sure this is needed?